lib/salestation/app/input_verification.rb

```diff
@@ -7,10 +7,10 @@ def verify_input(schema)
         -> (request) do
           result = schema.call(request.input)
           if result.success?
-            request.replace_input(result.output)
+            request.replace_input(result.to_h)
           else
             Deterministic::Result::Failure(
-              Errors::InvalidInput.new(errors: result.errors, hints: result.hints)
+              Errors::InvalidInput.new(errors: result.errors.to_h, hints: result.hints.to_h)
```

Comment:
I think we may want to make this compatible with the older version as we have done with dry-types https://github.com/salemove/salestation/pull/49/commits/7b835c6f743b68a34be8d5c670417a6615972fe5

the upgrade is not easy and other ruby services still use the old version of dry validation

Reply:
glia-errors shows an example of the dry-validation version switching. https://github.com/salemove/glia-errors/blob/master/ruby/lib/glia/errors.rb#L12-L19

Reply:
Let me check which services are still on the old version.

Reply:
Added switching https://github.com/salemove/salestation/compare/86d30a08aa975c8002eebeac4081716828280fc9..282ff143a3cdc4367f4d41697dea384b8a184312